calendar/Package.md

```diff
@@ -0,0 +1,20 @@
+Connects to Microsoft Outlook Calendar from Ballerina
+
+### Package overview
+The `outlook.calendar` is a [Ballerina](https://ballerina.io/) connector for Outlook Calendar.
```

Comment:
Let's use the connector name. 
Microsoft Outlook Calendar

Reply:
Changed. /Thanks